settings.gradle.kts

```diff
@@ -82,6 +82,7 @@ include(
     ":utils:testing-ui-activity",
     ":utils:testing-unit-common",
     ":utils:multiplatform",
+    ":utils:viewmodel",
```

Comment:
I'm not sure this belongs to utils, perhaps something like `android-node-viewmodel` ?

Reply:
I'd say utils is the best place for now, I wouldn't open a new top-level name for it.

Reply:
since it's not an KMP artifact I'd mention `android`